modules/ROOT/partials/configuration/a11ychecker_filter_issue.adoc

```diff
@@ -40,7 +40,26 @@ tinymce.init({
   plugins: 'a11ychecker',
   toolbar: 'a11ycheck',
   a11ychecker_filter_issue: (issue) => {
-    return issue.element.nodeName.toLowerCase() !== 'table';
+	  return !(issue.element.nodeName.toLowerCase() === 'table' || issue.severity !== 'error');
+  }
+});
+----
+
+=== Example: Using `+a11ychecker_filter_issue+` to filter images with empty alternative text from the Accessibility Checker I1 rule
+
+The callback function in the following example will only return `false` for any issues with `'I1'` as the `+'id'+` image elements with an empty `+'alt+'` attribute, otherwise the issue won't be filtered out. This implementation can be useful as allowing images to have empty alternative text can be another method of applying the `+role="presentation"+` attribute to mark an image as `+decorative+`.
```

Comment:
```suggestion
The callback function in the following example will only return `false` for any issues with `'I1'` as the `+'id'+` image elements with an empty `+'alt+'` attribute, otherwise the issue won’t be filtered out. This implementation can be useful as allowing images to have empty alternative text can be another method of applying the `+role="presentation"+` attribute to mark an image as `+decorative+`.
```